docs/posts/into-snowflake/index.qmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "Snow IO: loading data from other DBs into Snowflake"
+author: "Phillip Cloud"
+error: false
+date: "2024-02-23"
+categories:
+  - blog
+  - snowflake
+  - io
+  - productivity
+---
+
+## Recap
+
+We've [blogged about Snowflake IO before](../snowflake-io/index.qmd), in the
+context of getting local files into Snowflake as fast as possible.
+
+In this post, we'll show how to query another system and insert query results into
+Snowflake chunk by chunk.
+
+## Setup
+
+### Connect to your non-Snowflake system
+
+We'll connect to a postgres database running locally in a container. You
+should be able to swap in your own connection details as needed.
+
+```{python}
+from ibis.interactive import *  # <1>
+
+pg_con = ibis.connect("postgres://postgres:postgres@localhost/postgres")
+```
+
+1. Import Ibis for maximum productivity in interactive analysis.
+
+We'll use a test dataset that contains some baseball batting statistics.
+
+Ibis provides that example data, so we can dump that into postgres.
+
+
+```{python}
+pg_batting = pg_con.create_table(
+    "batting",
+    ibis.examples.Batting.fetch().to_pandas(),  # <1>
+    temp=True,  # <2>
+)
+```
+
+1. Yep, I'm using pandas here!
+2. Use a temporary table to avoid cluttering up the database.
+
+### Connect to Snowflake
+
+```{python}
+import os
+
+snow_con = ibis.connect(os.environ["SNOWFLAKE_URL"])  # <1>
+```
+
+1. Set the `SNOWFLAKE_URL` environment variable to your Snowflake connection
+   string, which will look like `snowflake://user:pass@account/database/schema?warehouse=my_warehouse`.
+
+## Profit
+
+### Construct an Ibis expression from the postgres data
+
+Let's build an Ibis expression based on the `batting` table in our postgres database.
+
+```{python}
+pg_batting
+```
+
+We can compute the average RBI per year per team.
```

Comment:
Can we add either a link to what RBI is or a "(Runs Batted In)" for people that might have no clue about baseball.